components/calendar/src/rata_die.rs

```diff
@@ -0,0 +1,57 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use core::ops::{Add, Mul, Sub};
+
+use crate::helpers::div_rem_euclid64;
+
+/// The *Rata Die*, or *R.D.*, or `fixed_date`: number of days since January 1, 1 CE.
+///
+/// See: <https://en.wikipedia.org/wiki/Rata_Die>
+///
+/// Keep this class INTERNAL because it shouldn't be possible to construct a RataDie
+/// except from a date that is in range of one of the official calendars.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub(crate) struct RataDie(pub i64);
+
+impl RataDie {
+    pub fn div_rem_euclid64(self, divisor: i64) -> (i64, i64) {
```

Comment:
nit: no need to call this 64, i think, we only have the 64 in the other one to disambiguate

Reply:
I deleted this function because it wasn't being used.